src/SignalR/common/SignalR.Common/src/Protocol/HubMethodInvocationMessage.cs

```diff
@@ -50,7 +50,14 @@ protected HubMethodInvocationMessage(string? invocationId, string target, object
     {
         if (string.IsNullOrEmpty(target))
         {
-            throw new ArgumentNullException(nameof(target));
+            if (target is null)
```

Comment:
Those three places are in projects that currently does not use any of the shared source. So using the throw helper would be somewhat more than simply including it. I assumed that they were kept separate from the shared source for a reason.

But maybe it would be better to make the shared source available here?

Reply:
> But maybe it would be better to make the shared source available here?

Yep, I don't see anything consequential with including the shared source in these projects for consistency's sake but let me know if you run into trouble with it.

Reply:
I have included it naively, but now there are build errors (as I also experienced when building locally). It is unclear to me what I have missed to include as compared to those places where it does work.

Reply:
The issue seems to be happening in projects that target netstandard or netfx, where nullability attributes are not defined in the runtime.

We have some build logic that includes stub definitions of these attributes for projects that target the two frameworks _and_ have nullability enabled (see [this chunk of MSBuild](https://github.com/dotnet/aspnetcore/blob/442d656e920d6122c16d703c56df3b4aa0a4ee22/eng/targets/CSharp.Common.targets#L111-L132)). The projects that have this issue don't enable nullability so this build logic doesn't execute.

We can try explicitly including the `<Compile Include="$(SharedSourceRoot)Nullable\NullableAttributes.cs" />` source in the affected projects.